goemail.py

```diff
@@ -1,137 +1,68 @@
-
 import webapp2
 from google.appengine.api import mail
 from google.appengine.api import users
 from google.appengine.api.taskqueue import taskqueue
 
-import band
 import gig
 import member
 import assoc
 import logging
 import re
-
 import pickle
 
 from webapp2_extras import i18n
-from webapp2_extras import jinja2
 from webapp2_extras.i18n import gettext as _
 
-SENDER_EMAIL = 'gigomatic.superuser@gmail.com'
+_admin_email_address = 'Gig-o-Matic <gigomatic.superuser@gmail.com>'
```

Comment:
Oh, the comment dropped the brackets. Is:

`SMTPSenderRefused(550, "5.1.0 <Gig-o-Matic <gigomatic.superuser@gmail.com>> size=2739 invalid address 'Gig-o-Matic <gigomatic.superuser@gmail.com>'", <email.header.Header instance at 0x10d3c2200>)`